data-service/src/main/resources/application-static.yaml

```diff
@@ -30,7 +30,7 @@ spring:
 akka:
   kafka:
     producer.topics:
-      output-topic.partitions: 1
+      success-topic.partitions: 1
```

Comment:
have references to this topic been renamed? I havent seen any other output >> success changes

Reply:
This was a lingering issue that was only noticed when trying to start up the data-service as there is no "output" topic associated with this service.